plugins/essif-lab_contactform7/src/Application/activator.php

```diff
@@ -1,9 +1,13 @@
 <?php
 
+namespace TNO\ContactForm7\Application;
+
+use TNO\ContactForm7\Implementation\Essif_Lab_Contactform7_Logic;
+
 class Essif_Lab_contactform7_Activator {
     public function activate() {
 
-        require_once plugin_dir_path( __FILE__ ) . '../Implementation/class-essif-lab_contactform7-logic.php';
+        require_once plugin_dir_path( __FILE__ ) . '../Implementation/logic.php';
```

Comment:
Met PSR4 worden alle klassen automatisch ingeladen door de autoloader in de bootstrap. Dit is dan dus niet meer nodig. Hier heb je dan al genoeg aan alleen de `use` op regel 5.